src/java.ts

```diff
@@ -24,6 +24,54 @@ declare global {
             }
         }
         namespace util {
+            class Scanner {
+                constructor(source: io.File);
+                constructor(source: io.File, charsetName: string);
+                constructor(source: string);
+            
+                close(): void;
+                findInLine(pattern: string): string | null;
+                findWithinHorizon(pattern: string, horizon: number): string | null;
+                hasNext(): boolean;
+                hasNext(pattern: string): boolean;
+                hasNextBigDecimal(): boolean;
+                hasNextBigInteger(): boolean;
+                hasNextBigInteger(radix: number): boolean;
+                hasNextBoolean(): boolean;
+                hasNextByte(): boolean;
+                hasNextByte(radix: number): boolean;
+                hasNextDouble(): boolean;
+                hasNextFloat(): boolean;
+                hasNextInt(): boolean;
+                hasNextInt(radix: number): boolean;
+                hasNextLine(): boolean;
+                hasNextLong(): boolean;
+                hasNextLong(radix: number): boolean;
+                hasNextShort(): boolean;
+                hasNextShort(radix: number): boolean;
+                next(): string;
+                next(pattern: string): string;
+                nextBigInteger(): BigInteger;
+                nextBigInteger(radix: number): BigInteger;
```

Comment:
The `BigInteger` type is not defined in this file yet. We'll either need to add an interface for it, or comment out these methods until we decide to add them. Either option is fine by me!

Reply:
I will remove the method for now. Seems like the safest option.